tpl_user_menu.php

```diff
@@ -12,20 +12,24 @@
 
 if (! empty($_SERVER['REMOTE_USER'])):
 
-$use_gravatar = bootstrap3_conf('useGravatar');
+$avatar_conf = bootstrap3_conf('userAvatar');
+$use_avatar = ($avatar_conf != 'noAvatar');
 
-if ($use_gravatar) {
-  $gravatar_img_small = ml(get_gravatar($INFO['userinfo']['mail'], 30).'&.jpg', array('cache' => 'recache', 'w' => 30, 'h' => 30));
-  $gravatar_img       = ml(get_gravatar($INFO['userinfo']['mail'], 64).'&.jpg', array('cache' => 'recache', 'w' => 64, 'h' => 64));
+if ($avatar_conf == 'Gravatar') {
+  $avatar_img_small = ml(get_gravatar($INFO['userinfo']['mail'], 30).'&.jpg', array('cache' => 'recache', 'w' => 30, 'h' => 30));
+  $avatar_img       = ml(get_gravatar($INFO['userinfo']['mail'], 64).'&.jpg', array('cache' => 'recache', 'w' => 64, 'h' => 64));
+} elseif ($avatar_conf == 'localAvatar') {
+  $avatar_img_small = '../../wiki/data/media/user/'.$_SERVER['REMOTE_USER'].'.jpg?nocache';
+  $avatar_img       = '../../wiki/data/media/user/'.$_SERVER['REMOTE_USER'].'.jpg?nocache';
```

Comment:
../../wiki/[...] won't work: it works in my website since I put every DokuWiki file inside a "wiki" folder. I don't know how to automatically get the "data/media" url, so this has to be fixed!